src/main.rs

```diff
@@ -169,60 +170,63 @@ impl Application {
 
     async fn on_text(&self, message: Message) -> Result<(), AudioPlayerError> {
         let msg = message.text;
-        if msg.starts_with("!") {
-            let tokens = msg[1..].split_whitespace().collect::<Vec<_>>();
+        if msg.starts_with("poke") {
+            let tokens = msg.split_whitespace().collect::<Vec<_>>();
+
+            let args = Command::from_iter_safe(&tokens);
+            match args {
+                Ok(v) => self.on_command(v).await?,
+                Err(e) => self.send_message(&format!("\n{}", e.message)),
```

Comment:
`error!` log?

Reply:
An error over here means that the user entered an invalid command, I don't think that user errors have to be logged.